src/rhsmlib/facts/cleanup.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from rhsmlib.facts import collector
+
+log = logging.getLogger(__name__)
+
+
+class CleanupCollector(collector.FactsCollector):
+    no_uuid_platforms = ['powervm_lx86', 'xen-dom0', 'ibm_systemz']
```

Comment:
I've seen this in 3 places. Can we please consolidate to 1? Honest question: can this logic go in python-rhsm?